test/test_tensordict.py

```diff
@@ -1178,6 +1178,21 @@ def test_pad(self, td_name, device):
         with pytest.raises(RuntimeError):
             pad(td, [0])
 
+    def test_reshape(self, td_name, device):
+        td = getattr(self, td_name)(device)
+        td_reshape = td.reshape(td.shape)
+        assert isinstance(td_reshape, TensorDict)
+        assert td_reshape.shape.numel() == td.shape.numel()
```

Comment:
Why compare `numel` when in this case we expect `td_reshape.shape == td.shape` no?